assets/js/components/Root/index.js

```diff
@@ -62,7 +62,7 @@ export default function Root( { children, registry, viewContext = null } ) {
 }
 
 Root.propTypes = {
-	children: PropTypes.node.isRequired,
```

Comment:
This looks like it can be reverted as there there should never be a case where we render `Root` without children.

Reply:
We have the `Root` element without children here: https://github.com/google/site-kit-wp/blob/9fe10cb8c8214391288ba078e5fa4520cca52f36/assets/js/googlesitekit-idea-hub-post-list.js#L61-L66